js_modules/dagster-ui/packages/ui-core/src/runs/QueuedRunCriteriaDialog.tsx

```diff
@@ -0,0 +1,164 @@
+      canOutsideClickClose
+      canEscapeKeyClose
+      onClose={onClose}
+    >
+      <DialogBody>
+        <Table style={{borderRight: `1px solid ${Colors.keylineDefault()}`}}>
+          <tbody>
+            {priority ? (
+              <tr>
+                <td>Priority</td>
+                <td>
+                  <Mono>{priority}</Mono>
+                </td>
+              </tr>
+            ) : null}
+            {maxConcurrentRuns !== undefined ? (
+              <tr>
+                <td>Max concurrent runs</td>
+                <td>
+                  <Mono>{maxConcurrentRuns}</Mono>
+                </td>
+              </tr>
+            ) : null}
+            {runTagLimits?.length ? (
+              <tr>
+                <td>Tag concurrency limits:</td>
+                <td>
+                  {runTagLimits.map((limit, i) => (
+                    <div style={{overflow: 'auto', paddingBottom: 10}} key={`tagLimit:${i}`}>
+                      <StructuredContentTable cellPadding="0" cellSpacing="0">
+                        <tbody>
+                          <tr>
+                            <td style={{width: 80, fontSize: 12}}>
+                              {limit.value !== undefined ? 'Tag' : 'Tag key'}
+                            </td>
+                            <td>
+                              <Tag interactive>
+                                {limit.value !== undefined
+                                  ? `${limit.key}=${limit.value}`
+                                  : limit.key}
+                              </Tag>
+                            </td>
+                          </tr>
+                          <tr>
+                            <td style={{width: 80, fontSize: 12}}>Limit</td>
+                            <td>{limit.limit}</td>
+                          </tr>
+                        </tbody>
+                      </StructuredContentTable>
+                    </div>
+                  ))}
+                </td>
+              </tr>
+            ) : null}
+            {runIsOpConcurrencyLimited ? (
+              <tr>
+                <td>
+                  <Box flex={{direction: 'row', alignItems: 'center'}}>
+                    <Box margin={{right: 4}}>Root concurrency keys</Box>
+                    <Tooltip
+                      placement="bottom"
+                      content="Op/asset concurrency limits are set on all of the initial steps in this run. This run will not start until there are available slots for at least one step"
+                    >
+                      <Icon name="info" color={Colors.accentGray()} />
+                    </Tooltip>
+                  </Box>
+                </td>
+                <td>
+                  {run.rootConcurrencyKeys!.map((key, i) =>
+                    runQueueConfig ? (
+                      <Tag interactive key={`rootConcurrency:${i}`}>
+                        <Link to={`/concurrency?key=${key}`}>{key}</Link>
```

Comment:
love that this links to the key - viewers will not be able to access the page though right?  wonder if we should build a read-only view of the page (not blocking)